WordPress/Classes/ViewRelated/User Profile Sheet/UserProfileSheetViewController.swift

```diff
@@ -0,0 +1,138 @@
+class UserProfileSheetViewController: UITableViewController {
+
+    // MARK: - Properties
+
+    private let user: RemoteUser
+
+    // MARK: - Init
+
+    init(user: RemoteUser) {
+        self.user = user
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - View
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTable()
+        registerTableCells()
+    }
+
+}
+
+// MARK: - DrawerPresentable Extension
+
+extension UserProfileSheetViewController: DrawerPresentable {
+
+    var collapsedHeight: DrawerHeight {
+        if traitCollection.verticalSizeClass == .compact {
+            return .maxHeight
+        } else {
+            return .contentHeight(300)
```

Comment:
Super nitpicky, and probably just a question of preference, but if you like we could do away with the else clause and just have the `return .contentHeight(300)`.

Reply:
Busted copying & pasting! 😄 Thank you!